### PR TITLE
Language bar and selector - Frontend

### PR DESCRIPTION
**Pull request by cemalokten**
#### Issues this PR relates to : 

- https://github.com/yalla-coop/cost-of-living-support/issues/53
- https://github.com/yalla-coop/cost-of-living-support/issues/72

### CODE REVIEW

File: client/src/components/Language/LanguageSelector/index.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+  en: 'English',
+  es: 'Spanish',
+  bm: 'Bambara',
+  as: 'French',
+  gs: 'Swedish',
+  vs: 'Japanese',
+  am: 'Bambara',
+  qm: 'German',
+  am: 'Russian',
+  lm: 'Italian',
```
Comment (fadeomar) [inline client/src/components/Language/LanguageSelector/index.js:26]: duplicate key am

**Review by fadeomar — CHANGES_REQUESTED** [review]
- [ ] the code is not running, please make sure that everything is working as expected 
image

**Comment by cemalokten** [comment]
@fadeomar Fixed this, when the COL version has been re-reviewed ill create a new patch to add the changes to this PR.

File: client/src/components/Icon/icons/German.js [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+    <rect opacity="0.01" width="24" height="24" fill="#D8D8D8" />
+    <defs>
+      <filter
+        id="filter0_d_0_55"
+        x="0"
+        y="9.14285"
+        width="24"
+        height="5.71429"
+        filterUnits="userSpaceOnUse"
+        color-interpolation-filters="sRGB"
```
Comment (fadeomar) [inline client/src/components/Icon/icons/German.js:55]:
```suggestion
        colorInterpolationFilters="sRGB"
```

File: client/src/components/Icon/icons/German.js [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+        />
+      </filter>
+      <filter
+        id="filter1_d_0_55"
+        x="0"
+        y="14.8571"
+        width="24"
+        height="5.71429"
+        filterUnits="userSpaceOnUse"
+        color-interpolation-filters="sRGB"
```
Comment (fadeomar) [inline client/src/components/Icon/icons/German.js:88]: use camelCase

File: client/src/components/Icon/icons/Urdu.js [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+    >
+      <rect
+        x="0.25"
+        y="0.25"
+        width="27.5"
+        height="19.5"
+        rx="1.75"
+        fill="white"
+        stroke="white"
+        stroke-width="0.5"
```
Comment (fadeomar) [inline client/src/components/Icon/icons/Urdu.js:36]: use camelCase

File: client/src/components/Icon/icons/Urdu.js [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+      />
+    </mask>
+    <g mask="url(#mask0_0_446)">
+      <path
+        fill-rule="evenodd"
+        clip-rule="evenodd"
```
Comment (fadeomar) [inline client/src/components/Icon/icons/Urdu.js:42]: use camelCase

File: client/src/components/Icon/icons/Urdu.js [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+        d="M28 0H8V20H28V0Z"
+        fill="#0A632F"
+      />
+      <path
+        fill-rule="evenodd"
+        clip-rule="evenodd"
```
Comment (fadeomar) [inline client/src/components/Icon/icons/Urdu.js:48]: use camelCase

File: client/src/components/Icon/index.js.rej [hunk 7]
```diff
@@ -0,0 +1,49 @@
+diff a/client/src/components/Icon/index.js b/client/src/components/Icon/index.js	(rejected hunks)
```
Comment (fadeomar) [inline client/src/components/Icon/index.js.rej:1]: this file does not suppose to be pushed on the PR

**Review by fadeomar — CHANGES_REQUESTED** [review]
- [x] each child in the language selector component needs a key
image
  - [x] please make sure to use camel case for the properties names in SVG's, there are many errors in the console because of that 
image
  - [ ] when click at any flag n he language selector component it throw error : type error handleHide is not a function 
image

**Pull request merged**